### ISSUE TITLE
[Swift] Access modifiers are omitted when generating a single file output with a namespace

### ISSUE THREAD

**Issue by gsabran** [apollographql/apollo-tooling#1449]
**Intended outcome:**

I am generating Swift code using a namespace. The output looks like 

```swift
public enum MyNameSpace {
    struct Hero: GraphQLFragment {
        public var name: String
        public var age: Int
    }
}
```

while it should be

```swift
public enum MyNameSpace {
    public struct Hero: GraphQLFragment {
        public var name: String
        public var age: Int
    }
}
```

As far as I can tell this is a regression introduced by https://github.com/apollographql/apollo-tooling/pull/1241. This change should only apply to multi file outputs that use public extensions of the namespace.

This issue prevents the usage of the generated objects outside of the exact module where they are defined.

If this is indeed the issue, the required package version in [apollo-ios](https://github.com/apollographql/apollo-ios/blob/master/scripts/check-and-run-apollo-cli.sh)  should be bumped once it is fixed.

Let me know if I can provide more information to help diagnose!

**Comment by designatednerd** [comment]
Argh! Thanks for the heads up, I'll get on this.

**Comment by designatednerd** [comment]
This is fixed in `2.17`, and will be part of iOS `0.14.0`
